staging/cse/windows/azurecnifunc.ps1

```diff
@@ -43,9 +43,11 @@ function Set-AzureCNIConfig
         [Parameter(Mandatory=$true)][bool]
         $IsDualStackEnabled,
         [Parameter(Mandatory=$false)][bool]
-        $IsAzureCNIOverlayEnabled
+        $IsAzureCNIOverlayEnabled,
+        [Parameter(Mandatory=$false)][bool]

```

Comment:
we'll need to release the changes to this file first as a new windows CSE script package before we can make the proposed change to kuberneteswindowssetup.ps1

cc @timmy-wright

Reply:
Thanks! updated release message. warmly ping @timmy-wright for a review.